src/rx-query.ts

```diff
@@ -521,6 +521,10 @@ export class RxQueryBase<
     }
 
     enablePersistentQueryCache(backend: QueryCacheBackend) {
+        if (this._persistentQueryCacheBackend) {
```

Comment:
Oh… this happened when re-rendering I assume?

Reply:
yeah i'm not fully sure yet, something like that